core/app/components/CopyCurrentUrlButton.tsx

```diff
@@ -1,10 +1,17 @@
 "use client";
 
+import dynamic from "next/dynamic";
+
 import type { CopyButtonProps } from "ui/copy-button";
 import { CopyButton } from "ui/copy-button";
 
 type Props = Omit<CopyButtonProps, "value">;
 
-export const CopyCurrentUrlButton = (props: Props) => {
+const CopyCurrentUrlButtonBase = (props: Props) => {
 	return <CopyButton value={window.location.href} {...props} />;
 };
+
+// necessary in order to disable SSR, as window is not available on the server
+export const CopyCurrentUrlButton = dynamic(() => Promise.resolve(CopyCurrentUrlButtonBase), {
+	ssr: false,
+});
```

Comment:
did this bc we can no longer define dynamic imports with `ssr: false` in server components.